frontend/components/dashboard/DashboardValidatorManagementModal.vue

```diff
@@ -563,21 +579,21 @@ const inputValidator = ref('')
 
             <template #bc-table-footer-left>
               <div
-                v-if="maxValidatorsPerDashboard"
                 class="left"
               >
                 <div
                   class="labels"
-                  :class="{ premiumLimit }"
+                  :class="{ 'premium-limit': hasReachedLimit }"
                 >
                   <span>
-                    <BcFormatNumber
-                      :value="totalValidators"
-                      default="0"
-                    /> /
-                    <BcFormatNumber
-                      :value="maxValidatorsPerDashboard"
-                      default="0"
+                    <BcFormatAmount
+                      :value="overview?.balances.total!"
```

Comment:
issue: do not disable typescript

```suggestion
                      :value="overview?.balances.total ?? '0'"
```

Reply:
This was wip, thanks for the reminder. Done